Scripts/Batch_simulator.m

```diff
@@ -6,7 +6,9 @@
 c = input("Enter Case: ");
 des = input("Enter Set Descriptor: ","s");
 x =strcat("16k-participants",int2str(c),des);
-h = input("Number of Trials");
+h = input("Number of Trials:");
+ACDR_fulfillment = [1,h];

```

Comment:
hmmm if you are trying to pre define the size, this won't do chuck HAHA kasi [1,5] lang literally this. Try zeros(1,h)